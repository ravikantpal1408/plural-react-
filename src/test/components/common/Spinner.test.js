import React from 'react';
import Spinner from '../../../components/common/Spinner';
import renderer from 'react-test-renderer';

describe('Match snapshot', () => {
  it('should match with spnapshot ', () => {
    const wrapper = new renderer.create(<Spinner />);
    expect(wrapper).toMatchSnapshot();
  });
});
