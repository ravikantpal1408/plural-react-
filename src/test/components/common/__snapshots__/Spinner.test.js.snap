// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match snapshot should match with spnapshot  1`] = `
<div
  className="loader"
>
  Loading...
</div>
`;
